### ISSUE TITLE
Azure data studio 1.35.1-1647367587 blank window and freezes my Debian 10

### ISSUE THREAD

**Issue by sebflaesch** [microsoft/azuredatastudio#18874]
- Azure Data Studio Version: 1.35.1-1647367587
- OS Version: 
$ uname -a
Linux toro 4.19.0-20-amd64 # 1 SMP Debian 4.19.235-1 (2022-03-17) x86_64 GNU/Linux
$ cat /etc/debian_version 
10.12

Steps to Reproduce:

1. Install azuredatastudio (1.35.1-1647367587) .deb package
2. Start azure data studio => blank window, then OS freezes and have to force power off!

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by Charles-Gagnon** [comment]
@abist The "linked" issue was because he had `#1` in the text, which Github converted into an issue link. It has nothing to do with the issue itself.

He mentions he's using 1.35.1 as the version of ADS

**Comment by kburtram** [comment]
@sebflaesch I installed a clean Deb 10.12 VM and ADS 1.35.1 from Deb file.  The uname and debian version match what's provided in issue.  I wasn't able to repro in that test.

Are you seeing any error messages anywhere that could help determine what's going on?  Any other context that may help repro the issue?  Does this occur every time you launch ADS?

**Comment by sebflaesch** [comment]
Thank you Karl for considering this.

What I saw yesterday after re-installing and starting ADS is a blank window, and a top command showed firefox-esr using 100% CPU ...
Trying to kill -9 the firefox-esr or azuredatastudio processes => no way... 

I have quite a few packages installed so maybe you could try to install some of them and see if it conflicts?

Here is the dpkg-query -l list...
[package-list.txt](https://github.com/microsoft/azuredatastudio/files/8439989/package-list.txt)

Seb

**Comment by sebflaesch** [comment]
FYI, just checking shared lib dependencies:

sf@toro:~$ ldd -r /usr/share/azuredatastudio/azuredatastudio 
	linux-vdso.so.1 (0x00007ffe869d8000)
	libffmpeg.so => /usr/share/azuredatastudio/libffmpeg.so (0x00007f512ca19000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f512c9f2000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f512c9d1000)
	libgobject-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0 (0x00007f512c97b000)
	libglib-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 (0x00007f512c85c000)
	libxshmfence.so.1 => /usr/lib/x86_64-linux-gnu/libxshmfence.so.1 (0x00007f512c65a000)
	libgio-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libgio-2.0.so.0 (0x00007f512c49a000)
	libnss3.so => /usr/lib/x86_64-linux-gnu/libnss3.so (0x00007f512c34c000)
	libnssutil3.so => /usr/lib/x86_64-linux-gnu/libnssutil3.so (0x00007f512c31a000)
	libsmime3.so => /usr/lib/x86_64-linux-gnu/libsmime3.so (0x00007f512c2eb000)
	libnspr4.so => /usr/lib/x86_64-linux-gnu/libnspr4.so (0x00007f512c2aa000)
	libatk-1.0.so.0 => /usr/lib/x86_64-linux-gnu/libatk-1.0.so.0 (0x00007f512c281000)
	libatk-bridge-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libatk-bridge-2.0.so.0 (0x00007f512c24a000)
	libdbus-1.so.3 => /lib/x86_64-linux-gnu/libdbus-1.so.3 (0x00007f512c1f7000)
	libdrm.so.2 => /usr/lib/x86_64-linux-gnu/libdrm.so.2 (0x00007f512c1e4000)
	libgdk_pixbuf-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0 (0x00007f512c1be000)
	libgtk-3.so.0 => /usr/lib/x86_64-linux-gnu/libgtk-3.so.0 (0x00007f512bacd000)
	libgdk-3.so.0 => /usr/lib/x86_64-linux-gnu/libgdk-3.so.0 (0x00007f512b9d1000)
	libpango-1.0.so.0 => /usr/lib/x86_64-linux-gnu/libpango-1.0.so.0 (0x00007f512b984000)
	libcairo.so.2 => /usr/lib/x86_64-linux-gnu/libcairo.so.2 (0x00007f512b864000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f512b6e1000)
	libX11.so.6 => /usr/lib/x86_64-linux-gnu/libX11.so.6 (0x00007f512b5a0000)
	libXcomposite.so.1 => /usr/lib/x86_64-linux-gnu/libXcomposite.so.1 (0x00007f512b39d000)
	libXdamage.so.1 => /usr/lib/x86_64-linux-gnu/libXdamage.so.1 (0x00007f512b398000)
	libXext.so.6 => /usr/lib/x86_64-linux-gnu/libXext.so.6 (0x00007f512b184000)
	libXfixes.so.3 => /usr/lib/x86_64-linux-gnu/libXfixes.so.3 (0x00007f512af7e000)
	libXrandr.so.2 => /usr/lib/x86_64-linux-gnu/libXrandr.so.2 (0x00007f512ad73000)
	libexpat.so.1 => /lib/x86_64-linux-gnu/libexpat.so.1 (0x00007f512ad36000)
	libxcb.so.1 => /usr/lib/x86_64-linux-gnu/libxcb.so.1 (0x00007f512ad0c000)
	libxkbcommon.so.0 => /usr/lib/x86_64-linux-gnu/libxkbcommon.so.0 (0x00007f512acca000)
	libgbm.so.1 => /usr/lib/x86_64-linux-gnu/libgbm.so.1 (0x00007f512acb8000)
	libasound.so.2 => /usr/lib/x86_64-linux-gnu/libasound.so.2 (0x00007f512abb7000)
	libatspi.so.0 => /usr/lib/x86_64-linux-gnu/libatspi.so.0 (0x00007f512ab83000)
	libcups.so.2 => /usr/lib/x86_64-linux-gnu/libcups.so.2 (0x00007f512aaf0000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f512aad6000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f512a916000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f513548c000)
	libffi.so.6 => /usr/lib/x86_64-linux-gnu/libffi.so.6 (0x00007f512a90a000)
	libpcre.so.3 => /lib/x86_64-linux-gnu/libpcre.so.3 (0x00007f512a896000)
	libgmodule-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libgmodule-2.0.so.0 (0x00007f512a890000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f512a872000)
	libmount.so.1 => /lib/x86_64-linux-gnu/libmount.so.1 (0x00007f512a813000)
	libselinux.so.1 => /lib/x86_64-linux-gnu/libselinux.so.1 (0x00007f512a5eb000)
	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007f512a5cf000)
	libplc4.so => /usr/lib/x86_64-linux-gnu/libplc4.so (0x00007f512a5c8000)
	libplds4.so => /usr/lib/x86_64-linux-gnu/libplds4.so (0x00007f512a5c3000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f512a5b9000)
	libsystemd.so.0 => /lib/x86_64-linux-gnu/libsystemd.so.0 (0x00007f512a518000)
	libpangocairo-1.0.so.0 => /usr/lib/x86_64-linux-gnu/libpangocairo-1.0.so.0 (0x00007f512a506000)
	libXi.so.6 => /usr/lib/x86_64-linux-gnu/libXi.so.6 (0x00007f512a2f6000)
	libcairo-gobject.so.2 => /usr/lib/x86_64-linux-gnu/libcairo-gobject.so.2 (0x00007f512a2ea000)
	libwayland-cursor.so.0 => /usr/lib/x86_64-linux-gnu/libwayland-cursor.so.0 (0x00007f512a2e0000)
	libwayland-egl.so.1 => /usr/lib/x86_64-linux-gnu/libwayland-egl.so.1 (0x00007f512a2db000)
	libwayland-client.so.0 => /usr/lib/x86_64-linux-gnu/libwayland-client.so.0 (0x00007f512a2ca000)
	libepoxy.so.0 => /usr/lib/x86_64-linux-gnu/libepoxy.so.0 (0x00007f512a196000)
	libharfbuzz.so.0 => /usr/lib/x86_64-linux-gnu/libharfbuzz.so.0 (0x00007f512a09b000)
	libpangoft2-1.0.so.0 => /usr/lib/x86_64-linux-gnu/libpangoft2-1.0.so.0 (0x00007f512a082000)
	libfontconfig.so.1 => /usr/lib/x86_64-linux-gnu/libfontconfig.so.1 (0x00007f512a03c000)
	libfreetype.so.6 => /usr/lib/x86_64-linux-gnu/libfreetype.so.6 (0x00007f5129f80000)
	libXinerama.so.1 => /usr/lib/x86_64-linux-gnu/libXinerama.so.1 (0x00007f5129f79000)
	libXcursor.so.1 => /usr/lib/x86_64-linux-gnu/libXcursor.so.1 (0x00007f5129f6c000)
	libthai.so.0 => /usr/lib/x86_64-linux-gnu/libthai.so.0 (0x00007f5129f61000)
	libfribidi.so.0 => /usr/lib/x86_64-linux-gnu/libfribidi.so.0 (0x00007f5129f44000)
	libpixman-1.so.0 => /usr/lib/x86_64-linux-gnu/libpixman-1.so.0 (0x00007f5129e9e000)
	libpng16.so.16 => /usr/lib/x86_64-linux-gnu/libpng16.so.16 (0x00007f5129e65000)
	libxcb-shm.so.0 => /usr/lib/x86_64-linux-gnu/libxcb-shm.so.0 (0x00007f5129e5e000)
	libxcb-render.so.0 => /usr/lib/x86_64-linux-gnu/libxcb-render.so.0 (0x00007f5129e4f000)
	libXrender.so.1 => /usr/lib/x86_64-linux-gnu/libXrender.so.1 (0x00007f5129c45000)
	libXau.so.6 => /usr/lib/x86_64-linux-gnu/libXau.so.6 (0x00007f5129a41000)
	libXdmcp.so.6 => /usr/lib/x86_64-linux-gnu/libXdmcp.so.6 (0x00007f512983b000)
	libwayland-server.so.0 => /usr/lib/x86_64-linux-gnu/libwayland-server.so.0 (0x00007f5129824000)
	libgssapi_krb5.so.2 => /usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2 (0x00007f51297d7000)
	libgnutls.so.30 => /usr/lib/x86_64-linux-gnu/libgnutls.so.30 (0x00007f512962a000)
	libavahi-common.so.3 => /usr/lib/x86_64-linux-gnu/libavahi-common.so.3 (0x00007f512961c000)
	libavahi-client.so.3 => /usr/lib/x86_64-linux-gnu/libavahi-client.so.3 (0x00007f5129609000)
	libblkid.so.1 => /lib/x86_64-linux-gnu/libblkid.so.1 (0x00007f51295b2000)
	liblzma.so.5 => /lib/x86_64-linux-gnu/liblzma.so.5 (0x00007f512958a000)
	liblz4.so.1 => /usr/lib/x86_64-linux-gnu/liblz4.so.1 (0x00007f512956b000)
	libgcrypt.so.20 => /lib/x86_64-linux-gnu/libgcrypt.so.20 (0x00007f512944d000)
	libgraphite2.so.3 => /usr/lib/x86_64-linux-gnu/libgraphite2.so.3 (0x00007f5129420000)
	libuuid.so.1 => /lib/x86_64-linux-gnu/libuuid.so.1 (0x00007f5129415000)
	libdatrie.so.1 => /usr/lib/x86_64-linux-gnu/libdatrie.so.1 (0x00007f512940b000)
	libbsd.so.0 => /usr/lib/x86_64-linux-gnu/libbsd.so.0 (0x00007f51293f1000)
	libkrb5.so.3 => /usr/lib/x86_64-linux-gnu/libkrb5.so.3 (0x00007f5129311000)
	libk5crypto.so.3 => /usr/lib/x86_64-linux-gnu/libk5crypto.so.3 (0x00007f51292dd000)
	libcom_err.so.2 => /lib/x86_64-linux-gnu/libcom_err.so.2 (0x00007f51292d5000)
	libkrb5support.so.0 => /usr/lib/x86_64-linux-gnu/libkrb5support.so.0 (0x00007f51292c6000)
	libkeyutils.so.1 => /lib/x86_64-linux-gnu/libkeyutils.so.1 (0x00007f51292bf000)
	libp11-kit.so.0 => /usr/lib/x86_64-linux-gnu/libp11-kit.so.0 (0x00007f5129190000)
	libidn2.so.0 => /usr/lib/x86_64-linux-gnu/libidn2.so.0 (0x00007f5129171000)
	libunistring.so.2 => /usr/lib/x86_64-linux-gnu/libunistring.so.2 (0x00007f5128fed000)
	libtasn1.so.6 => /usr/lib/x86_64-linux-gnu/libtasn1.so.6 (0x00007f5128dd8000)
	libnettle.so.6 => /usr/lib/x86_64-linux-gnu/libnettle.so.6 (0x00007f5128da0000)
	libhogweed.so.4 => /usr/lib/x86_64-linux-gnu/libhogweed.so.4 (0x00007f5128d67000)
	libgmp.so.10 => /usr/lib/x86_64-linux-gnu/libgmp.so.10 (0x00007f5128ce4000)
	libgpg-error.so.0 => /lib/x86_64-linux-gnu/libgpg-error.so.0 (0x00007f5128cc1000)

**Comment by sebflaesch** [comment]
Tested removal of $HOME/.azuredatastudio directory... not better.

Also wanted to strace /usr/share/azuredatastudio/azuredatastudio but could not colled the output...

I did a ps -aef | grep azure to see what runs but could not collect the output.

BTW top showed a grep command using 100% cpu ...?!?!

Could not even reboot properly so powered off.

For now:

root@toro:~# dpkg -r azuredatastudio
(Reading database ... 273713 files and directories currently installed.)
Removing azuredatastudio (1.35.1-1647367587) ...
Processing triggers for desktop-file-utils (0.23-4) ...
Processing triggers for gnome-menus (3.31.4-3) ...
Processing triggers for mime-support (3.62) ...

root@toro:~# dpkg -P azuredatastudio
(Reading database ... 269440 files and directories currently installed.)
Purging configuration files for azuredatastudio (1.35.1-1647367587) ...
